assets/state-container-toolkit/0200_role.yaml

```diff
@@ -12,3 +12,9 @@ rules:
   - use
   resourceNames:
   - privileged
+- apiGroups:
```

Comment:
I was not aware of the toolkit inspecting daemonsets? Was thsi change intentional?

Reply:
Is this because the validator runs here too? I thought we changed the logic to just look for the ready file?

Reply:
Yes, it is because the validator runs here as an init container. We still need to run the driver validation here since the operator-validator pod runs with the `nvidia` runtime class, and thus, needs the toolkit pod to run first before it can run and start performing any validations.